app/experimenter/templates/experiments/field_inline_help_link.html

```diff
@@ -0,0 +1,22 @@
+<!-- this template exists to provide form-inline html with a
```

Comment:
Okay I see what you did here, this is sensible.  There's a lot of fields where the help text is only a link, like:

https://github.com/mozilla/experimenter/blob/master/app/experimenter/experiments/constants.py#L619-L706

and if we're going to introduce a new form field template that expects the help text to just be an outgoing link instead of a drop down we should probably switch everything to use it, but that's a lot of changes that don't belong in this PR.  

So I would say either move this template out into a separate issue/pr, and migrate all the fields that need it to it, or land the template as part of this change and file a separate issue to move all the other fields over to using it, whichever feels saner to you.

This is one of those cases where this is kind of a separate change that supports this PR but if we reverted this PR, we wouldn't want to also lose this change, becuase it's just a generally better way to organize this.  So I'll leave it to you to decide whether to keep it here and file the separate issue to change other fields to use it or do all that in a separate PR.  

Either way good call this is a good change 👍

Reply:
Hey sooo.... I'm not totally getting what you're saying here. The fields you are highlighting seem kind of different. The review links, like `REVIEW_BUGZILLA_HELP_TEXT` and `REVIEW_ENGINEERING_HELP_TEXT`.. aren't those used to display the word help after the checkbox in the required signoff section?  In that case the formatting is decided by Django and just comes from declaring `help_text` on the form... right? I can't see how what we are doing in that section relates to what we're doing here. And in the case of those other constants, like `RISK_INTERNAL_ONLY_HELP_TEXT` and `RISK_PARTNER_RELATED_HELP_TEXT`.. those are using the radio widget.. final formatting also defined by saying `help_text` on the form... I guess I am not seeing why those need to be changed or exactly how it relates to needing an inline_field with help link instead of help drop down.

Reply:
It's okay nevermind me, I'll have to look at it again and figure out if there's something we can do to make things more consistent, but this is fine.